perlmod/Fink/VirtPackage.pm

```diff
@@ -1465,10 +1469,13 @@ END
 		$options{debug} && printf STDERR " - program %s... %s\n", $file, -x $file ? "found" : "missing!";
```

Comment:
This block is not required with Xcode 10.1, this was only for Xcode 10.0, so we may want to omit this as it can comical other parts of builds and should just ban Xcode 10.0 instead IMHO.

Reply:
I don't understand. The lines in "this block" are between 4 and 13 years old, and have nothing to do with Xcode 10.0...

Reply:
Perhaps the question is what you mean with "this block"? To me, that wold refer to the line (1465 / 1469) you are commenting on, plus the block it is contained in. But perhaps you rather mean the code following it? But that, too, is fairly old.

BTW, I am surprised so see that this fix is only now needed -- from colleagues, I was under the impression that `/usr/include` had been removed already before 10.14 ?!

Reply:
I couldn't select the diff block but I meant that entire block of the patch isn't required, /usr/include was fixed in Xcode 10.1, it was only broken in 10.0

If we want to support the other way LOTS of packages need to be changed including pkgconfig and any package the has NoSet flags will need to change, plus ton os other random changes to configure flags all over.

Reply:
I have Xcode 10.1, and there is no `/usr/include`. To get `/usr/include`, one can install a special package, but I think we need to find a way to work without that -- most people won't know how to install it. Well, we could try to convince them to install it, but Apple quite strongly suggests that one should avoid using it...

Reply:
@fingolfin - Are you sure `/usr/include` is still not in Xcode CLTools 10.1? I do have `/usr/include` on my Mac with that version.

Reply:
On my computer, I had CLtools 10.0, and have updated to 10.1. I still don't have `/usr/include`. Same on my work computer - though there I have run `installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /` to add it. But we can't assume that users will know about this workaround.

Reply:
I had Xcode 10.0 and CLITools 10.0 and updated both to 10.1 and I have /usr/include afterwards

Maybe both are required to fix it?  Maybe we could add the detection of /usr/include to fink and tell them to run that command?